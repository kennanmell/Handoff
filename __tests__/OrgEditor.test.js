import 'react-native';
import React from 'react';
import OrganizationEditor from '../View/OrgEditor';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(
    	<OrganizationEditor
    		onSave={ () => {
    		}}
    	/>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});