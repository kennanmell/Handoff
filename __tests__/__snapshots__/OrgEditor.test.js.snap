exports[`test renders correctly 1`] = `
<View
  style={
    Object {
      "padding": 10,
    }
  }>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#612e5f",
        "fontSize": 15,
        "fontWeight": "bold",
        "margin": 0,
        "textAlign": "left",
      }
    }>
    Name
  </Text>
  <TextInput
    defaultValue="Hope Shelter"
    onChangeText={[Function]}
    style={
      Object {
        "height": 40,
      }
    } />
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#612e5f",
        "fontSize": 15,
        "fontWeight": "bold",
        "margin": 0,
        "textAlign": "left",
      }
    }>
    Location
  </Text>
  <TextInput
    defaultValue="1516 Brooklyn Ave"
    onChangeText={[Function]}
    style={
      Object {
        "height": 40,
      }
    } />
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#612e5f",
        "fontSize": 15,
        "fontWeight": "bold",
        "margin": 0,
        "textAlign": "left",
      }
    }>
    Mission Statement
  </Text>
  <TextInput
    defaultValue="Providing a space for those in need"
    multiline={true}
    numberOfLines={4}
    onChangeText={[Function]}
    style={
      Object {
        "height": 90,
      }
    } />
  <View
    accessibilityComponentType={undefined}
    accessibilityLabel={undefined}
    accessibilityTraits={undefined}
    accessible={true}
    hitSlop={undefined}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Array [
        Object {
          "backgroundColor": "transparent",
        },
        undefined,
      ]
    }
    testID={undefined}>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#00cc99",
          "fontSize": 18,
          "fontWeight": "bold",
          "margin": 0,
          "textAlign": "left",
        }
      }>
      Save
    </Text>
  </View>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    onPress={[Function]}
    style={
      Object {
        "color": "#00cc99",
        "fontSize": 18,
        "fontWeight": "bold",
        "margin": 0,
        "textAlign": "left",
      }
    }>
    Close
  </Text>
</View>
`;
